woocommerce/woocommerce-admin
# Jetpack Backup note - change time range and add an image

This changes the Jetpack Backup note that's currently shown in the inbox to:
* Start showing after 3 days instead of 1 week. It should show between 3 and 28 days after Woo install.
* Include an image.

### Screenshots
<img width="763" alt="Screen Shot 2022-02-10 at 5 07 55 PM" src="https://user-images.githubusercontent.com/690843/153670441-72983201-8f81-434b-96a4-7a4a2d7b2709.png">

### Detailed test instructions:

#### Setup

* Set up a WordPress site with the Jetpack, WooCommerce, and [WooCommerce Admin Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper/releases) plugins installed and activated.
* If not using a local dev environment, you can create a zip file for this branch of the plugin by running `npm run test:zip`, then install & activate that on your site.

#### Testing the note

This PR changes the time range that the note should be shown after a WooCommerce install, so we'll mainly be changing the install date to confirm that the note either shows or doesn't.

1. Login to MySQL on your test site. If using the [Docker & wp-env method](https://github.com/woocommerce/woocommerce-admin/blob/main/docker/wc-admin-wp-env/README.md#woocommerce-admin-docker-setup-with-wp-env) you can do that by first finding the MySQL port with the command:

```
npm run wp-env-mysql-port
```
 	
Then logging in with (change the value of `-P` to your port number):

```
mysql -u root -h 127.0.0.1 -P 54836 -p wordpress
```
And enter the password: `password`.

2. Delete any existing rows for the note:

```
delete from wp_wc_admin_notes where name = 'wc-admin-marketing-jetpack-backup' limit 1;
```

3. From wp-admin, go to **Tools > WCA Test Helper**.
4. Under the `Tools` tab click `run` for `Update WC installation timestamp` and update the date to **1 day ago**.
5. Click `run` for `Run wc_admin_daily job`.
6. Go to **WooCommerce > Home**.
7. Confirm that you **do not** see the Jetpack Backup note.
8. Repeat steps 2-5, this time changing the install date to **10 days ago** ([here's](https://www.timeanddate.com/date/dateadd.html) a handy date calculator).
9. Under **WooCommerce > Home** confirm that you **do** see the Jetpack Backup note and that it appears with an image as shown in the screenshot.
10. Repeat steps 2-5 again, this time changing the install date to **35 days ago**.
11. Under **WooCommerce > Home** confirm that you **do not** see the Jetpack Backup note.
